### PR TITLE
feat(plasma-tokens, plasma-tokens-utils): Add shadow token generate

### PR DESCRIPTION
**Pull request by neretin-trike**
Добавлена оптимизация кода и частично переработаны механизмы генерации новых тем. 

Добавлены токены теней в json уже готовых схем.

Обновлена `src` директория с новыми добавленными токенами теней, которые теперь можно использовать в веб-компонентах.

Проставлены "автоматические комментарии" для файлов старой типографики, которые генерируются скриптом.

В [рамках обсуждения](https://github.com/salute-developers/plasma/pull/494#pullrequestreview-1405595915) удалена генерация файлов `values` из тем для новых схем.

:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.496.5013213279.swift)  
[color_sberHealth_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.496.5013213279.kt)  
[color_sberHealth_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.496.5013213279.ts)  
[color_sbermarket_business_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.496.5013213279.swift)  
[color_sbermarket_business_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.496.5013213279.kt)  
[color_sbermarket_business_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.496.5013213279.ts)  
[color_sbermarket_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.496.5013213279.swift)  
[color_sbermarket_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.496.5013213279.kt)  
[color_sbermarket_metro_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.496.5013213279.swift)  
[color_sbermarket_metro_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.496.5013213279.kt)  
[color_sbermarket_metro_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.496.5013213279.ts)  
[color_sbermarket_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.496.5013213279.ts)  
[color_sbermarket_selgros_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.496.5013213279.swift)  
[color_sbermarket_selgros_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.496.5013213279.kt)  
[color_sbermarket_selgros_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.496.5013213279.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.496.5013213279.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.496.5013213279.kt)  
[color_sbermarket_wlbusiness_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.496.5013213279.ts)  
[color_sberprime_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.496.5013213279.swift)  
[color_sberprime_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.496.5013213279.kt)  
[color_sberprime_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.496.5013213279.ts)  
[shadow_sbermarket_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.496.5013213279.ts)  
[typo_mage_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.496.5013213279.swift)  
[typo_mage_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.496.5013213279.kt)  
[typo_mage_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.496.5013213279.ts)  
[typo_plasma_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.496.5013213279.swift)  
[typo_plasma_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.496.5013213279.kt)  
[typo_plasma_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.496.5013213279.ts)  
[typo_ruler_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.496.5013213279.swift)  
[typo_ruler_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.496.5013213279.kt)  
[typo_ruler_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.496.5013213279.ts)  
[typo_sage_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.496.5013213279.swift)  
[typo_sage_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.496.5013213279.kt)  
[typo_sage_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.496.5013213279.ts)  
[typo_sbermarket_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.496.5013213279.swift)  
[typo_sbermarket_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.496.5013213279.kt)  
[typo_sbermarket_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.496.5013213279.ts)  
[typo_soulmate_ios-swift--canary.496.5013213279.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.496.5013213279.swift)  
[typo_soulmate_kotlin--canary.496.5013213279.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.496.5013213279.kt)  
[typo_soulmate_react-native--canary.496.5013213279.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.496.5013213279.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.211.0-canary.496.5013213279.0
  npm install @salutejs/plasma-core@1.116.0-canary.496.5013213279.0
  npm install @salutejs/plasma-hope@1.211.0-canary.496.5013213279.0
  npm install @salutejs/plasma-icons@1.145.0-canary.496.5013213279.0
  npm install @salutejs/plasma-temple@1.160.0-canary.496.5013213279.0
  npm install @salutejs/plasma-tokens-b2b@1.19.0-canary.496.5013213279.0
  npm install @salutejs/plasma-tokens-b2c@0.28.0-canary.496.5013213279.0
  npm install @salutejs/plasma-tokens-web@1.34.0-canary.496.5013213279.0
  npm install @salutejs/plasma-tokens@1.51.0-canary.496.5013213279.0
  npm install @salutejs/plasma-ui@1.192.0-canary.496.5013213279.0
  npm install @salutejs/plasma-web@1.211.0-canary.496.5013213279.0
  npm install @salutejs/plasma-cy-utils@0.58.0-canary.496.5013213279.0
  npm install @salutejs/plasma-sb-utils@0.114.0-canary.496.5013213279.0
  npm install @salutejs/plasma-tokens-utils@0.25.0-canary.496.5013213279.0
  # or 
  yarn add @salutejs/plasma-b2c@1.211.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-core@1.116.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-hope@1.211.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-icons@1.145.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-temple@1.160.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-tokens-b2b@1.19.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-tokens-b2c@0.28.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-tokens-web@1.34.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-tokens@1.51.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-ui@1.192.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-web@1.211.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-cy-utils@0.58.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-sb-utils@0.114.0-canary.496.5013213279.0
  yarn add @salutejs/plasma-tokens-utils@0.25.0-canary.496.5013213279.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-496/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-496/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-496/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-496/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-496/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-496/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-496/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-496/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-496/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-496/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-496/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-496/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-496/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-496/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-496/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-496/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: utils/plasma-tokens-utils/src/generation.ts [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+    TokenType,
+    TypoSystem,
+    GeneratedFiles,
+    TypoStyles,
+    GeneratedTokenType,
+} from './types';
 import { escapeValue } from './utils';
 
-type GeneratedTokenType = 'value' | 'css';
+// TODO: зарефакторить этот файл (или удалить в plasma 2.0)
```
Comment (Yakutoc) [inline utils/plasma-tokens-utils/src/generation.ts:16]: @neretin-trike 
Тогда давай добавим issue что бы не забыть об этом TODO?

File: utils/plasma-tokens-native/package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
     "keywords": ["plasma", "tokens", "ios", "react-native", "kotlin"],
     "scripts": {
         "prepare": "npm run build",
-        "prebuild": "mkdir -p data && cp -R node_modules/@salutejs/plasma-tokens/data/* data",
+        "prebuild": "rm -rf ./build ./properties ./data && npm run mkdir",
+        "mkdir": "mkdir -p data && cp -R node_modules/@salutejs/plasma-tokens/data/* data",
```
Comment (Yakutoc) [inline utils/plasma-tokens-native/package.json:15]: [Уточнение]

@neretin-trike 
Я правильно понимаю тут идет рекурсивная запись?

**Comment by Yakutoc** [comment]
@neretin-trike Нужно провести мастер-класс по созданию тем )) для лучшего понимания логики кода

File: utils/plasma-tokens-utils/src/utils/other.ts [hunk 3]
```diff
@@ -15,5 +16,11 @@ export const escapeValue = <T = string | number>(value: T) => {
...
 export const join = (...args: (string | undefined)[]) => args.filter(Boolean).join('-');
 
-export const getCSSVariableName = (key: string, fromData = false) =>
-    `--${DESIGN_SYSTEM_PREFIX}-${key}`.replace(fromData ? 'plasma-colors-' : '', '');
+export const getCSSVariableName = (key: string, withDesignPrefix = true) =>
+    withDesignPrefix ? `--${DESIGN_SYSTEM_PREFIX}-${key}` : `--${key}`;
+
+export const getBoxShadow = (shadows: Array<Shadow>) =>
+    shadows.map(
+        ({ offset, blurRadius, spreadRadius, color }) =>
```
Comment (Yakutoc) [inline utils/plasma-tokens-utils/src/utils/other.ts:24]: [minor]

`{ offset:  { x, y }  }`

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+/**
+ * Преобразует объект токенов в CSS Properties object.
+ * @param {TokenDataGroup<TokenType>} tokens Объект токенов
+ * @param {GeneratedTokenType} type Тип выводимого токена
+ * @param {string} name Имя токена
+ * @param {GeneratedTokenMode} mode Вид выводимого токена
+ * @return {string}
+ */
+export const generateTokenSet = ({ tokens, type, mode }: GenerateTokens) =>
+    Object.entries(tokens || {}).reduce(
```
Comment (Yakutoc) [inline utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:93]: @neretin-trike А если проверку сделать для tokens? Если tokens будет empty зачем его через reduce пропускать?

```js
if (!tokens) {
  return '';
}
```

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    let result = '';
+
+    if (!Array.isArray(value)) {
+        return value;
+    }
+
+    const newValue = getBoxShadow((value as unknown) as Array<Shadow>);
+
+    if (type === 'css') {
+        result = newValue.map((val) => toCSSVarTokenWithValue(`${paramCase(name)}`, val, withPrefixDesign)).join(', ');
```
Comment (Yakutoc) [inline utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:37]: [Уточнение]

@neretin-trike А какие возможны значения для **name**?

`paramCase(name)`

File: utils/plasma-tokens-utils/src/utils/objectToCSSShadowVariables.ts [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import type { DataObject, Shadow } from '../types';
+
+import { join, getCSSVariableName, getBoxShadow } from './other';
+
+export const objectToCSSShadowVariables = (obj: DataObject) => {
+    const withPrefixDesign = false;
+
+    return Object.keys(obj)
+        .filter((key) => key !== 'modes' && key !== 'comment')
```
Comment (Yakutoc) [inline utils/plasma-tokens-utils/src/utils/objectToCSSShadowVariables.ts:9]: [Minor]

`key !== 'modes' && key !== 'comment'` => `!['modes', 'comment'].includes(key)`

File: utils/plasma-tokens-native/package.json (hunk 2 above)
Comment (neretin-trike) [reply utils/plasma-tokens-native/package.json:15]: не, тут я создаю папку `data`, а затем копирую в неё содержимое из пути `node_modules/@salutejs/plasma-tokens/data/`

Comment (Yakutoc) [reply utils/plasma-tokens-native/package.json:15]: Я про `cp -R` ) ну это так для моего общего понимания

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts (hunk 4 above)
Comment (neretin-trike) [reply utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:93]: это сделано прост для того, чтобы стрелочная функция была без оператора `return` (мне так больше нравится, т.к. кажется, что код чище). А reduce при `tokens = {}` всё-равно не сработает

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts (hunk 5 above)
Comment (neretin-trike) [reply utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:37]: в данном случае сюда прилетает имя самого токена, который указывается в json'e

File: utils/plasma-tokens-utils/src/utils/objectToCSSShadowVariables.ts (hunk 6 above)
Comment (neretin-trike) [reply utils/plasma-tokens-utils/src/utils/objectToCSSShadowVariables.ts:9]: не уверен, что в данном случае имеет смысл, т.к. переменных всего две, и кажется, что вызывать целый метод ради них немного излишне. Но если бы их бы больше, тогда да

**Review by Yakutoc — APPROVED** [review]

File: utils/plasma-tokens-native/package.json (hunk 2 above)
Comment (neretin-trike) [reply utils/plasma-tokens-native/package.json:15]: а, тут да

**Review by kayman233 — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

File: utils/plasma-tokens-utils/src/generation.ts (hunk 1 above)
Comment (Yeti-or) [reply utils/plasma-tokens-utils/src/generation.ts:16]: а удалить потому что это старый формат?

File: utils/plasma-tokens-native/package.json (hunk 2 above)
Comment (Yeti-or) [reply utils/plasma-tokens-native/package.json:15]: можно без звездочки 
а можно даже без создания файла:


```
"mkdir": "mkdir -p data && cp -R node_modules/@salutejs/plasma-tokens/data/* data",
 => 

"copyTokensData": "cp -R node_modules/@salutejs/plasma-tokens/data  data",
```

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+    if (typeof value !== 'string' && !(typeof value === 'object' && 'origin' in value)) {
+        return value;
+    }
+
+    const newValue = typeof value === 'object' ? value.origin : value;
+
+    if (type === 'css') {
+        result = toCSSVarTokenWithValue(`${paramCase(name)}`, newValue, withPrefixDesign);
+    }
+    if (type === 'value') {
```
Comment (Yeti-or) [inline utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:58]: мы их все ешё хотим генериить?

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+ * @param {GeneratedTokenType} type Тип выводимого токена
+ * @param {string} name Имя токена
+ * @param {GeneratedTokenMode} mode Вид выводимого токена
+ * @return {string}
+ */
+export const generateTokenSet = ({ tokens, type, mode }: GenerateTokens) =>
+    Object.entries(tokens || {}).reduce(
+        (acc, [name, token]) => `${acc}${generateToken({ token, type, name, mode })}\n`,
+        '',
+    );
```
Comment (Yeti-or) [inline utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:96]: тут всё так сложно получается что мне уже хочется написать шаблонизатор =/

File: utils/plasma-tokens-utils/src/utils/writeGeneratedToFS.ts [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+import { ROBO_COMMENT } from '../constants';
+import type { GeneratedFiles } from '../types';
+
+/**
+ * Запись нагенерированного в файловую систему.
+ * @param {string} dir Директория для записи (если не существует, то будет создана).
+ * @param {GeneratedFiles} generated Сгенерированный контент для записи.
+ * @param {string?} generateMessage Сообщение, которое добавляется сверху файла (по умолчанию ROBO_COMMENT).
+ */
+export const writeGeneratedToFS = (dir: string, generated: GeneratedFiles, generateMessage?: string) => {
```
Comment (Yeti-or) [inline utils/plasma-tokens-utils/src/utils/writeGeneratedToFS.ts:13]: `generateMessage?: string = ROBO_COMMENT)`

**Review by Yeti-or — APPROVED** [review]

File: utils/plasma-tokens-utils/src/generation.ts (hunk 1 above)
Comment (neretin-trike) [reply utils/plasma-tokens-utils/src/generation.ts:16]: ага, он генерит старые токены

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts (hunk 7 above)
Comment (neretin-trike) [reply utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:58]: вообще нет, но я этот функционал оставил "на всякий случай". Но могу в принципе и выпилить

File: utils/plasma-tokens-utils/src/generators/generateTokenSet.ts (hunk 8 above)
Comment (neretin-trike) [reply utils/plasma-tokens-utils/src/generators/generateTokenSet.ts:96]: да, по-хорошему бы надо придумать, как это можно упростить

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-496/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-496/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-496/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-496/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-496/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-496/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-496/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Pull request merged**